### ISSUE TITLE
Clips Window

### ISSUE THREAD

**Issue by Reaper10** [tenacityteam/tenacity-legacy#296]
pro tool clip window 
![Compact+in+the+Pro+Tools+Clip+List+Contextual+Menu](https://user-images.githubusercontent.com/8137561/126049203-035907f7-7ac9-47e7-8c3f-5be18ffc9425.jpg)

adobe addiction clip window 
![736433713c33bf4a9606aed3d3e1dee5](https://user-images.githubusercontent.com/8137561/126049354-bff5ce4c-129c-4fbe-b7a0-d4b10bcbed01.png)

ocen audio clip window 
![ocenaudio](https://user-images.githubusercontent.com/8137561/126049462-fedc5ee1-f9fc-4fb9-867f-7c562dd66239.jpg)

all pro have some from of a  clips window 
so a user can find all their materials in one place.

I think it should have  a search box.
 something like ocen audio 
![ocenaudio](https://user-images.githubusercontent.com/8137561/126050105-98f44ea1-1d26-4bca-86f8-6641827361a0.jpg)

i will be adding more info as I find it 

https://www.youtube.com/watch?v=48AiZFg9kKU
https://www.youtube.com/watch?v=LufIJQsmRCU

i will be making icons for this soon.

midi file icon 

audio file icon 

video file icon 

This would all so help to make tenacity less a destructive editor. After all you wouldn't be editing the original file.

**Comment by n0toose** [comment]
I'm not sure if I understand what you're proposing here.

**Comment by RoaddogLabs** [comment]
When audio segments are separated the result is called a “clip”.  For example if I have a voice over that’s four sentences and I make a cut at the end of each sentence it would create four clips.  I could then give those clips a meaningful name.  The side nav menu to which he references contains a listing of all the clips of that project.  Using that menu I can better identify and manage the clips.  I could move them around changing sentence order or do things like replace words or entire parts of songs.  Great feature.  I makes identifying and manipulating small segments of audio easier.

However, I don’t know how many using would need or even know how to use it.  @Reaper10 appears to be an advanced user based on his previous posts.  For me the feature would be helpful.  I’m not so sure if that translates to most or even many users but those of us using it professionally would benefit.  It could also have a benefit for the less experienced editors to show them a new workflow and teach more about audio editing methods.  One doesn’t know what they don’t know and this could expose them to another way to work.   It’s a standard feature in most pro level editors/DAWs.

ETA more about the workflow

**Comment by Reaper10** [comment]
https://www.youtube.com/watch?v=LufIJQsmRCU
https://www.youtube.com/watch?v=48AiZFg9kKU
will this help

**Comment by asperkraken** [comment]
@panos 
- @Reaper10  means a drop-down menu with list of files. Opening/browsing multiple files in Audacity by default requires launching another window. I believe they are asking for a way to browse files without going to Windows Explorer in-program (most DAWs have this feature). It's actually quite a pain, a legacy issue with Audacity. Am i phrasing this correctly?
-Seems a reasonable request as this could reduce memory usage and improve the fork's performance.

**Comment by nm17** [comment]
@asperkraken I agree, Tenacity needs this feature to expand it's functionality, although it's probably best to start doing it after our first release.

**Comment by Reaper10** [comment]
This would all so help to make tenacity less a destructive editor. After all you wouldn't be editing the original file.
